samples/replicatedentity-counter/dependency-reduced-pom.xml

```diff
@@ -0,0 +1,335 @@
+<?xml version="1.0" encoding="UTF-8"?>

```

Comment:
Should the `dependency-reduced-pom.xml` file be committed, or gitignored?

Reply:
it's generated by the maven plugin, I removed it again and added to gitignore